modules/ffmff/manifests/supernode.pp

```diff
@@ -318,6 +318,12 @@
       order  => 99,
       source => 'users',
       dest   => 'net:+exit';
+    'outgoing-mail':
+      source => 'users',
+      destination => 'net',
```

Comment:
nope; not this way.
`destination => ['net', 'ovpn'],`
maybe but not twice the same rule